tests/unit/test_amazon_estimator.py

```diff
@@ -63,6 +64,13 @@ def test_init(sagemaker_session):
     assert pca.num_components == 55
 
 
+def test_s3_init(sagemaker_session):
```

Comment:
I think all the tests you've added here should go on their own file. This way we have a better organized test suite. This existing file is meant to test the AmazonEstimator. you should create a file to test your new estimators. The content of the tests  is fine just split it into its own file.

Reply:
The test I wrote tests the AmazonS3Estimator, which is on the same module as AmazonEstimator, which is why I think that this belongs on this file. I have a serparate test for image classification tests.